packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -369,15 +370,18 @@ async function needMigrateToArmAndMultiEnv(ctx: CoreHookContext): Promise<boolea
   if (!fxExist) {
     return false;
   }
-
+  const parameterEnvFileName = parameterFileNameTemplate.replace(
+    "@envName",
```

Comment:
You can reuse `constants.EnvNamePlaceholder` from the api package

Reply:
thanks